react-props-and-expressions/my-app/src/index.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Fira Sans",
+    "Droid Sans",
+    "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    "Courier New",
+    monospace;
+}
+
+button {
+  height: 3rem;
+  width: 6rem;
+  border-radius: 0.5rem;
+}
+
+.buttonOne {
```

Comment:
I suggest you put these in CustomButton.css and import that file in CustomButton.js.